source/backend/api/Services/DocumentGenerationService.cs

```diff
@@ -74,33 +75,39 @@ public async Task<ExternalResult<FileDownload>> GenerateDocument(string template
 
             // TODO: This needs to retrieve by the passed template type. At this point that is not possible.
             PimsDocument document = _documentRepository.GetAllByDocumentType("CDOGS Template").LastOrDefault();
```

Comment:
This needs to be updated now that there are proper document types

Reply:
Hmm, I'll need to log a follow up story I guess, since for this story those changes aren't present yet.